src/gui/folderwizard.cpp

```diff
@@ -43,17 +43,17 @@
 
 namespace OCC {
 
-QString FormatWarningsWizardPage::formatWarnings(const QStringList &warnings) const
+QString FormatWarningsWizardPage::formatWarnings(const QStringList &warnings, bool error) const
 {
     QString ret;
     if (warnings.count() == 1) {
-        ret = tr("<b>Warning:</b> %1").arg(warnings.first());
+        ret = error ? tr("<b>Error:</b> %1").arg(warnings.first()) : tr("<b>Warning:</b> %1").arg(warnings.first());
     } else if (warnings.count() > 1) {
-        ret = tr("<b>Warning:</b>") + " <ul>";
+        QStringList w2;
         for (const auto &warning : warnings) {
-            ret += QString::fromLatin1("<li>%1</li>").arg(warning);
+            w2.append(QStringLiteral("<li>%1</li>").arg(warning));
         }
-        ret += "</ul>";
+        ret = error ? tr("<b>Error:</b><ul>%1</ul>").arg(w2.join(QString())) : tr("<b>Warning:</b><ul>%1</ul>").arg(w2.join(QString()));
```

Comment:
Seems redundant to the block above.

Reply:
In theory yes but I'm not an expert on rtl languages and whether there might be a case where the position of the argument needs to be changed.
Also this gives the translator more contexts.